testdata/includes_flatten_default/.gitignore

```diff
@@ -0,0 +1 @@
+file.txt
```

Comment:
Files should end with a line break.

The same applies to the next two files where they're also missing.